sbt

```diff
@@ -34,8 +34,8 @@
 
 set -o pipefail
 
-declare -r sbt_release_version="1.3.13"
```

Comment:
All of the changes here are from pasting in the most up-to-date version of `sbt-extras`.